assets/constants.ts

```diff
@@ -29,6 +29,22 @@ export default {
   // fragment in the controller is invalid.
   "DEFAULT_CLASSROOM_URL_FRAGMENT": "math",
 
+  "FEATURED_TRANSLATION_LANGUAGES": [
+    {"language_code": "pt", "explanation": "português (Portuguese)"},
```

Comment:
Wouldn't just the language codes be enough (and we call this FEATURED_TRANSLATION_LANGUAGE_CODES)? The descriptions should be retrieved from the existing languages dict, and for safety we should have a test that verifies that every language code in this list exists in the existing languages dict.

That way you don't have multiple sources of truth for the explanation.

Reply:
We can do that but please note that currently also we do not use the "explanation" as the "description". Just to explain it more, we fetch the description of the language code from the [constants.SUPPORTED_AUDIO_LANGUAGES](https://github.com/oppia/oppia/blob/develop/assets/constants.ts#L5355) and the "explanation" is used when the user hovers over the "i" button present over the language.
Currently, I have put in the same value as the "description" to the "explanation" because I was not sure what to put there.
Let me know if we plan to remove the "explanation" part.

I have added the test as requested in the "contributor_dashboard_test.py" file.

Thanks

Reply:
Got it, thanks for the info.
I will make sure to ask if I'm not sure.
Thanks